corpora/csip/metadata/amdsec/CSIP31/IP_18000_CSIP31_2/METS.xml

```diff
@@ -0,0 +1,89 @@
+  <!-- CSIP18 MUST - An identifier for the descriptive metadata section (dmdSec) used for referencing inside the package -->
+  <!-- CSIP18 is part of the CSIP17 test-case, because the XML schema for METS.xml require it - otherwise the METS.xml would no be valid according to the XML schema -->
+  <fileSec ID="ID_filesec">
+    <!-- CSIP59 there MUST be an ID -->
+    <!-- schemas -->
+    <fileGrp USE="Schemas" ID="ID_schemas">
+      <!-- CSIP60 there MUST be a fileGrp-element -->
+      <!-- CSIP64 there MUST be an USE-attribute -->
+      <!-- CSIP65 there MUST be an ID-attribute -->
+      <!-- METS.xsd -->
+      <file ID="ID_mets_xsd" MIMETYPE="application/xml" SIZE="133920" CREATED="2018-05-01T14:20:00"
+        CHECKSUM="4e9961dec3de72081e6142b28a437fb8" CHECKSUMTYPE="MD5">
+        <!-- CSIP66 there MUST be a file-element -->
+        <!-- CSIP67 there MUST be an ID-attribute -->
+        <!-- CSIP68 there MUST be an MIMETYPE-attribute -->
+        <!-- CSIP69 there MUST be an SIZE-attribute -->
+        <!-- CSIP70 there MUST be an CREATED-attribute -->
+        <!-- CSIP71 there MUST be an CHECKSUM-attribute -->
+        <!-- CSIP72 there MUST be an CHECKSUMTYPE-attribute -->
+        <FLocat LOCTYPE="URL" xlink:type="simple" xlink:href="schemas/mets.xsd"/>
+        <!-- CSIP76 there MUST be a FLocat-element -->
+        <!-- CSIP77 there MUST be a LOCTYPE-attribute -->
+        <!-- CSIP78 there MUST be a xlink:type-attribute with the value "simple" -->
+        <!-- CSIP79 there MUST be a xlink:href-attribute -->
+      </file>
+      <!-- XMLSchema.xsd -->
+      <file ID="ID_XMLSchema_xsd" MIMETYPE="application/xml" CREATED="2015-12-14T14:20:00"
+        CHECKSUM="94ed1a93ce3147d01bcb2fc1126255ed" CHECKSUMTYPE="MD5" SIZE="87677">
+        <FLocat LOCTYPE="URL" xlink:href="schemas/XMLSchema.xsd" xlink:type="simple"/>
+      </file>
+      <!-- xlink.xsd -->
+      <file ID="ID_xlink_xsd" MIMETYPE="application/xml" CREATED="2015-12-14T14:20:00"
+        CHECKSUM="90c7527e6d4d3c3a6247ceb94b46bcf5" CHECKSUMTYPE="MD5" SIZE="8322">
+        <FLocat LOCTYPE="URL" xlink:href="schemas/xlink.xsd" xlink:type="simple"/>
+      </file>
+      <!-- CSIPExtensionMETS.xsd -->
+      <file ID="ID_CSIPExtensionMETS.xsd" MIMETYPE="application/xml" CREATED="2018-12-14T14:20:00"
+        CHECKSUM="1a31b3aa3ae1e9b99e7a8b4618f3b485" CHECKSUMTYPE="MD5" SIZE="1673">
+        <FLocat LOCTYPE="URL" xlink:href="schemas/CSIPExtensionMETS.xsd" xlink:type="simple"/>
+      </file>
+    </fileGrp>
+  </fileSec>
+  <structMap TYPE="PHYSICAL" LABEL="CSIP StructMap" ID="StructmapID">
+    <!-- CSIP80 mets/structMap -->
+    <!-- CSIP81 mets/structMap/@TYPE The type attribute of the structural map (structMap) is set to value “PHYSICAL” from the vocabualry -->
+    <!-- CSIP82 mets/structMap/@LABEL The value must be “CSIP StructMap” -->
+    <!-- CSIP83 mets/structMap/@ID -->
+    <div ID="Structmap_Div_ID" LABEL="IP_18000_CSIP17_2">
```

Comment:
Wrong LABEL value (maybe not the biggest mistake)

Reply:
Fixed